rclcpp_components/src/component_container_isolated.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
Technically, the copyright can remain OSRF, and there may be (legal) reasons to prefer leaving it as-is for both the contributor and OSRF. But, there's precedence for allowing additional copyright holders in ROS 2 source code. I would defer to @clalancette and/or @tfoote for clarity.

Reply:
It's certainly not required to put your own name there, but it's also totally fine as long as the changes to the file are non-trivial or it's a new file (which is the case here).

So, putting yourself there is ok, and leaving it as-is is also ok. :)

Reply:
> So, putting yourself there is ok, and leaving it as-is is also ok. :)

Yes, exactly.  Since this is a non-trivial amount of new work, it is perfectly acceptable to make yourself the copyright holder.  In that case, you would change the line to be `// Copyright 2021 <my name here>`.  On the other hand, it is also perfectly fine to "assign" the copyright to the Open Source Robotics Foundation, which gives the foundation most of the legal power over the code.  Totally up to you which way you want to go.

(the usual disclaimer of "I am not a lawyer" and "this is not legal advice" applies here)